### PR TITLE
samples: moving autoML samples from nodejs-translate and refactored them

### LINKED ISSUE #200: Move automl samples to googleapis/nodejs-automl
We have `automl` samples sitting in this repo.  While they're samples about translation, they're unrelated to this npm module.  We should move these samples over to googleapis/nodejs-automl, so that they're tested as part of the release process of that module.

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Fixes https://github.com/googleapis/nodejs-translate/issues/200 🦕

### CODE REVIEW

File: samples/beta/cancel_operation.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  operationFullId = 'projects/YOUR_PROJECT_ID/locations/YOUR_LOCATIOIN/operations/OPERATION_ID'
```
Comment (JustinBeckwith) [inline samples/beta/cancel_operation.js:18]: This part of the code isn't really shown in the user facing snippets, and won't be used in our tests - so I don't think there's much value in defaulting the value.

File: samples/beta/cancel_operation.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const location = 'us-central1';
+  // const datasetId = 'YOUR_DATASET_ID';
+
+  // Imports the Google Cloud AutoML library
+  const {AutoMlClient} = require('@google-cloud/automl').v1beta1;
+
+  console.log(operationFullId);
```
Comment (JustinBeckwith) [inline samples/beta/cancel_operation.js:32]: That feels a little out of place

File: samples/beta/cancel_operation.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  const client = new AutoMlClient();
+
+  async function cancelOperation() {
+    // Construct request
+    const request = {
+      name: operationFullId,
+    };
+
+    client.operationsClient.cancelOperation({
+      name: operationFullId.toString(),
```
Comment (JustinBeckwith) [inline samples/beta/cancel_operation.js:43]: Huh, what format do you expect this in?  If it's a string, don't think you need to `toString()` it

File: samples/package.json [hunk 4]
```diff
@@ -17,6 +17,7 @@
   ],
   "dependencies": {
     "@google-cloud/automl": "^2.2.0",
+    "@google-cloud/translate": "^6.0.2",
```
Comment (JustinBeckwith) [inline samples/package.json:20]: huh, weird - don't think you need this, right?

**Review by JustinBeckwith — CHANGES_REQUESTED** [review]

File: samples/beta/cancel_operation.js (hunk 2 above)
Comment (munkhuushmgl) [reply samples/beta/cancel_operation.js:32]: ooops i was testing something, forgot to remove it

File: samples/package.json (hunk 4 above)
Comment (munkhuushmgl) [reply samples/package.json:20]: oh yea we do not need it, do not know what I was thinking

**Comment by munkhuushmgl** [comment]
@JustinBeckwith  friendly ping.

File: samples/test/automlTranslation.v1beta1.test.js [hunk 5]
```diff
@@ -1,129 +0,0 @@
-// Copyright 2018 Google LLC
```
Comment (JustinBeckwith) [inline samples/test/automlTranslation.v1beta1.test.js:1]: If the PR was limited to moving samples from translate over here, it's a little unclear why this is getting deleted

**Review by JustinBeckwith — APPROVED** [review]
LGTM, with just a little confusion on the deleted tests.

File: samples/test/automlTranslation.v1beta1.test.js (hunk 5 above)
Comment (munkhuushmgl) [reply samples/test/automlTranslation.v1beta1.test.js:1]: I found out most of these old samples were deleted in other languages, and deleted samples' region tags were not used anywhere in codesearch or doc.

note: these tests were not really written, resources and IDs are no where to be found. Tests are very complex. In Python and java, they are already refactored and deleted. Therefore, I deleted and refactored this test into samples/test/automlTranslation.test.js

**Pull request merged**